root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4136,46 +4141,82 @@ function doMatchSubtest(
     relationshipTypesByUuid[relUuid]?.find(function (s) {
       return s === expectedRelationshipType;
     });
-  st.equal(actualRelationshipType, expectedRelationshipType, 'Match ' + label + ' URL relationship type for ' + entityType + ' entities');
+  st.equal(
+    actualRelationshipType,
+    expectedRelationshipType,
+    'Match ' + label + ' URL relationship type for ' +
+    entityType + ' entities',
+  );
   previousMatchTests.push(entityType + '+' + url);
 }
 
 testData.forEach(function (subtest, i) {
   test('input URL [' + i + '] = ' + subtest.input_url, {}, function (st) {
     let tested = false;
     if (!subtest.input_url) {
-      st.fail('Test is invalid: "input_url" is missing: ' + JSON.stringify(subtest));
+      st.fail(
+        'Test is invalid: "input_url" is missing: ' + JSON.stringify(subtest),
+      );
       st.end();
       return;
     }
     if (subtest.input_entity_type) {
       if ('expected_relationship_type' in subtest) {
-        if (previousMatchTests.indexOf(subtest.input_entity_type + '+' + subtest.input_url) !== -1) {
-          st.fail('Match test is worthless: Duplication has been detected: ' + JSON.stringify(subtest));
+        if (previousMatchTests.indexOf(
+          subtest.input_entity_type + '+' + subtest.input_url,
+        ) !== -1) {
+          st.fail(
+            'Match test is worthless: Duplication has been detected: ' +
+            JSON.stringify(subtest),
+          );
         }
-        doMatchSubtest(st, subtest.input_entity_type, subtest.input_url, 'input', subtest.expected_relationship_type);
+        doMatchSubtest(
+          st,
+          subtest.input_entity_type,
+          subtest.input_url,
+          'input',
+          subtest.expected_relationship_type,
+        );
         tested = true;
       } else {
-        st.fail('Test is invalid: "input_entity_type" is specified without "expected_relationship_type".');
+        st.fail(
+          'Test is invalid: "input_entity_type" is specified ' +
+          'without "expected_relationship_type".',
+        );
         st.end();
         return;
       }
     } else if ('expected_relationship_type' in subtest) {
-      st.fail('Test is invalid: "expected_relationship_type" is specified without "input_entity_type".');
+      st.fail(
+        'Test is invalid: "expected_relationship_type" is specified ' +
+        'without "input_entity_type".',
+      );
```

Comment:
These strings should not be split, the new `tests/.eslintrc` probably allows that.